cypress/e2e/auth.cy.ts

```diff
@@ -251,6 +251,67 @@ describe('Authentication', () => {
         cy.contains('This pseudo is already used by another player');
     });
 
+    it('sign up and change password', () => {
+        const pseudo = 'test-' + randomString();
+        const password = 'test';
+
+        cy.visit('/signup');
+
+        const signupForm = () => cy.contains('h2', 'Create an account').closest('form');
+        signupForm().contains('Pseudo').click().type(pseudo);
+        signupForm().contains('Password').click().type(password);
+        signupForm().contains('button', 'Create account').click();
+
+        cy.visit('/settings');
```

Comment:
This makes a hard refresh, and in my case, it aborts an ajax request, and make an error that stop the test.

(`TypeError: NetworkError when attempting to fetch resource.`)

A solution would be to do like an user do (which is more like a functionnal test), clicking on links, which navigate through the vue app without reloading page.

```suggestion
        cy.get('.menu-top').contains(pseudo).click();
        cy.contains('Settings').click();
```